index.js

```diff
@@ -314,12 +316,15 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
 }
 
 // predicate-based error filter 
-function SkipedError ( error ) {
+function SkippedError ( error ) {
     return error.skip === true;
 }
 
-function errorHandler ( body )  {
+function errorHandler ( options, body )  {
     if ( body.error ) {
+        // Missing data is flagged by the error code 100
+        body.error.skip = options.ignoreMissing && body.error.code === 100
```

Comment:
Please convert this error code to a constant and add a link to the relevant documentation page referencing the error - https://developers.facebook.com/docs/marketing-api/error-reference

Also I think that the comment should use the same phrasing as the documentation. Error 100 means "missing parameters". If i understand it correctly, this doesn't mean that there is an object missing - rather that the request is missing some parameters.